src/Hotel.js

```diff
@@ -7,7 +7,7 @@ class Hotel {
     return this.bookings.filter(booking => booking.userID === userID)
   }
 
-  getPastBookings(today, userID, ) {
+  getPastBookings(today, userID) {
     const orderedBookings = this.getBookingsByID(userID).sort((a, b) => a.date - b.date);
     return orderedBookings.filter(booking => booking.date < today)
   }
```

Comment:
I like the idea of using another class Hotel to handle some of the work. Makes a lot of sense